lib/assessment.rb

```diff
@@ -5,7 +5,7 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    enum.max_by { |x| x.length }.to_i
```

Comment:
No, I think the question is asking for the longest thing, given whatever 'long' means in the context of that thing. Might be strings, for instance.